test/src/e2e.dynval/setup.ts

```diff
@@ -180,9 +180,10 @@ export async function createNodes(options: {
                 )
                 .sign({
                     secret: faucetSecret,
-                    seq: faucetSeq2 + i,
+                    seq: faucetSeq2,
```

Comment:
I've changed same line 🤔 How about this?
https://github.com/CodeChain-io/codechain/pull/1703/files#diff-2ad9cd6449c41e2610357eef55525900R190

Reply:
Looks good. It's so functional!